CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+- Support list jobs without default prefix *.
```

Comment:
`Support listing jobs ... ` might be better.

Reply:
> Thanks @tiantn - this works for me and addresses what I was after with #70

Thanks @dkelosky and @std4lqi!